ui/static/ui/js/listing_resources.jsx

```diff
@@ -1,7 +1,138 @@
-define('listing_resources', ['react', 'jquery', 'lodash'],
-  function (React, $, _) {
+define('listing_resources', ['react', 'jquery', 'lodash', 'utils'],
+  function (React, $, _, Utils) {
     'use strict';
 
+    var ICheckbox = Utils.ICheckbox;
+
+    var getImageFile = function(resourceType) {
+      if (resourceType === 'chapter') {
```

Comment:
I don't think there's a difference. In any case this code was already here, it was just moved up the page